Cod3rs_Growth/Coders_Growth.Web/wwwroot/services/Mensagens.js

```diff
@@ -0,0 +1,35 @@
+sap.ui.define([
+    "sap/m/MessageBox",
+
+], function (MessageBox,) 
+{
+    "use strict";
+
+    return {
+
+        confirmar: function (mensagemAviso, mensagemCancelado, functionRemover, id) {
+            return MessageBox.confirm(mensagemAviso, {
+                actions: [MessageBox.Action.YES, MessageBox.Action.CANCEL],
+                onClose: (acao) => {
+                    if (acao === MessageBox.Action.YES) {
+                        return functionRemover.apply(this, id);
+                    }
+                    
+                    return MessageBox.error(mensagemCancelado);
+                }
+            })
+        },
+
+        falhou: function (mensagem) {
```

Comment:
As funções de falha e aviso são muito parecidas, muda só um detalhe... Talvez desse pra criar um método genérico pra evitar repetição de código

Reply:
Deixei apenas a função de aviso, pois se for erro ou apenas uma informação o aviso se torna genérico